starboard/extension/extension_test.cc

```diff
@@ -582,8 +582,10 @@ TEST(ExtensionTest, MediaSettings) {
   }
 
   EXPECT_STREQ(extension_api->name, kExtensionName);
-  EXPECT_EQ(extension_api->version, 1u);
+  EXPECT_EQ(extension_api->version, 2u);
```

Comment:
Assume a partner implements this at version 1, this test will fail.  Is this expected?  @y4vor FYI